functions/Get-DbaAvailabilityGroup.ps1

```diff
@@ -90,7 +90,7 @@ function Get-DbaAvailabilityGroup {
 				if ($IsPrimary) {
 					$defaults = 'ComputerName','InstanceName','SqlInstance','Name as AvailabilityGroup','IsPrimary'
 					$value = $false
-					if ($serverName -eq $ag.PrimaryReplicaServerName) {
+					if ($ag.PrimaryReplicaServerName -eq $serverName) {
```

Comment:
The side of the operator does make a difference when dealing with implicit casts. The data type of the left side will be used to make the comparison. If you have $serverName first it will try to compare a DbaInstanceParameter object to a string, which will not match. If you put $ag.PrimaryReplicaServerName first it will cast DbaInstanceParameter to a string, which will invoke the class' implicit operator string function and return just the server name instead of the entire DbaInstanceParameter object.

Take a look at this screen shot for an example of how the order matters when dealing with objects of different types. Unfortunately I am on vacation now and don't readily have access to my server where I was having this issue. 

![image](https://user-images.githubusercontent.com/7840633/28741360-7f1cb740-73e2-11e7-84a5-b522dc2d5a9b.png)